proto/executor/v1/executor.proto

```diff
@@ -545,6 +553,8 @@ message ProcessTransactionResponseV2 {
     uint32 has_gasprice_opcode = 19;
     // Flag to indicate if opcode 'BALANCE' has been called
     uint32 has_balance_opcode = 20;
+    // Receipt status of the transaction, 1 = success, 0 = failure
+    uint32 status = 21;
```

Comment:
Shouldn't this be called statusV2 ?

Reply:
it is not needed since in inside the `ProcessTransactionResponseV2`